cyber/scheduler/policy/scheduler_choreography.cc

```diff
@@ -110,10 +110,18 @@ void SchedulerChoreography::CreateProcessor() {
     processors_.push_back(std::move(proc));
   }
 
+  if (task_pool_size_ > 0) {
+    ClassicContext::cr_group_[DEFAULT_GROUP_NAME];
+    ClassicContext::rq_locks_[DEFAULT_GROUP_NAME];
+    ClassicContext::mtx_wq_[DEFAULT_GROUP_NAME];
+    ClassicContext::cv_wq_[DEFAULT_GROUP_NAME];
+  }
```

Comment:
we can move 113:118 into the for loop below to make code look clean?

Reply:
just need init once，so put here